File: Java/Busqueda_secuencial.java
public class Busqueda_secuencial {
    public static void main(String[] args) {
        int[] arreglo = {15, 3, 42, 8, 23};
        int n = 5;
        int valor = 42;
        int resultado = -1;

        for (int i = 0; i < n; i++) {
            if (arreglo[i] == valor) {
                resultado = i;
                break;
            }
        }

        System.out.println(resultado);
    }
}

File: JavaScript/Busqueda_secuencial.js
let arreglo = [15, 3, 42, 8, 23];

function busSecuencial(valor) {
    for (let i = 0; i < arreglo.length; i++) {
        if (arreglo[i] === valor) {
            return i;
        }
    }
    return -1;
}

console.log(busSecuencial(42));